package.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
just wanna make sure we can really delete this file and it wasn't an accident

Reply:
Yeah, I think it's residual from trying to install webpacker. It can be safely deleted.